MaxText/maxtext_utils.py

```diff
@@ -178,14 +178,16 @@ def calculate_gemma2_tflops_training_per_device(config, total_ffn_flops, qkv_flo
   return attention_tflops, learnable_weight_tflops
 
 
-def calculate_gemma3_tflops_training_per_device(config, total_ffn_flops, qkv_flops, projection_flops, embedding_flops):
+def calculate_mixed_attention_tflops_training_per_device(
```

Comment:
Could you name in a way to indicate it's not just attention flops,
e.g. `calculate_mixed_attention_model_tflops...`
or calculate_tflops_for_mixed_attention_model_...` 
I know they are getting long

Reply:
Thanks Gagik! Let's discuss offline? I am a little bit confused :)